docs/security/threat-model.md

```diff
@@ -8,44 +8,76 @@ nav_order: 2
 # Threat Model
 
 The design goals for Authelia is to protect access to applications by collaborating with reverse proxies to prevent
-attacks coming from the edge of the network. This document gives an overview of what Authelia is protecting against but some
-of those points are also detailed in [Security Measures](./measures.md).
+attacks coming from the edge of the network. This document gives an overview of what Authelia is protecting against.
+Some of these ideas are expanded on or otherwise described in [Security Measures](./measures.md).
 
 ## General assumptions
 
-Authelia is considered to be running within a trusted network and it heavily relies on the first level of security provided by reverse proxies. It's very important that you take time configuring your reverse proxy properly to get all the authentication benefits brought by Authelia.
-Some general security tweaks are listed in [Security Measures](./measures.md) to give you some ideas.
+Authelia is considered to be running within a trusted network and it heavily relies on the first level of security 
+provided by reverse proxies. It's very important that you take time configuring your reverse proxy properly to get all 
+the authentication benefits brought by Authelia.
+Some general security tweaks are listed in [Security Measures](./measures.md#additional-proxy-protection-measures) to 
+give you some ideas.
```

Comment:
```suggestion
and give you some ideas.

Reply:
This suggested `and` looks out of place I don't think it's necessary.